docs/plans/2026-04-21-event-otm-gate-plan.md

```
@@ -0,0 +1,696 @@
+
+## Test Plan
+
+- [x] New unit tests in `packages/dashboard/src/services/AutoSignalExecutor.eventOTMGate.test.ts`
+- [x] Full `pnpm test` passes on the dashboard package
+- [x] `pnpm build` passes
+- [ ] Post-deploy SQL verification (above)
+
+Related to #117.
+EOF
+)"
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)